cmd/aws-janitor/main.go

```diff
@@ -143,3 +183,24 @@ func markAndSweep(opts resources.Options, region string) error {
 
 	return nil
 }
+
+func pushMetricBeforeExit(pusher *push.Pusher, startTime time.Time, jobType bool, exitCode int) {
+	// Set the status of the job
+	status := "failed"
+	if exitCode == 0 {
+		status = "success"
+	}
+
+	// Set the type of the job to report the metric
+	job := "sweep"
+	if jobType {
+		job = "clean"
+	}
+
+	duration := time.Since(startTime).Seconds()
+	cleaningTimeHistogram.WithLabelValues(job, status).Observe(duration)
```

Comment:
also, slight preference for using `With(prometheus.Labels{"key": value, ...})` over `WithLabelValues`, as the latter feels a bit more brittle.

Reply:
ok!